### ISSUE TITLE
NTR: Bone marrow: hemophagocytosis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9118]
New term request Bone marrow: hemophagocytosis (elthat_100424151947): Phagocytosis by macrophages of erythrocytes, leukocytes, platelets, and their precursors in bone marrow and other tissues, macrophage activation syndrome, MAS, HLH; erythrophagocytosis  UMLS: 3807249

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000980
- label: Bone marrow hemophagocytosis
- parents: [Hemophagocytosis (HP:0012156)](https://hpo.jax.org/app/browse/term/HP:0012156)
- pmid: 
- def: Phagocytosis of erythrocytes, lymphocytes or other hematopoietic precursors by histiocytes or macrophages observed in the bone marrow.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9118
- synonyms:
